src/components/views/rooms/LinkPreviewWidget.tsx

```diff
@@ -116,6 +116,11 @@ export default class LinkPreviewWidget extends React.Component<IProps> {
             </div>;
         }
 
+        // Don't show anything if we're about to render nothing
+        if (!img && !p["og:description"] && !p["og:title"]) {
+            return null;
+        }
```

Comment:
This needs filtering out at the LinkPreviewGroup otherwise `Show N more` might lie about how many it can actually show